src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -53,13 +54,11 @@ const exportFormats: ExportFormat[] = [
     name: 'CSV',
     description: 'For import into a spreadsheet',
   },
-
-  // TODO Enable these formats when implemented
-  // {
-  //   value: 'html',
-  //   name: 'HTML',
-  //   description: '',
-  // },
+  {
+    value: 'html',
+    name: 'HTML',
+    description: 'For rich text representation of the annotations',
```

Comment:
I choose this  description as I don't see how one would import the HTML anywhere. Instead I think this suggests you can open the HTML and see a rich text document that you can then use however you like.

Reply:
Now that we have both TXT and HTML,  I think the important distinction to communicate is plain vs rich text. In Slack I suggested something like this:

<img width="404" alt="Text formats" src="https://github.com/hypothesis/client/assets/2458/de307bf9-0796-49a2-872e-c5b05cdefde8">

For the purposes of this PR, you could use "For import into word processors as plain text" and "For import into word processors as rich text" as the descriptions of "TXT" and "HTML" respectively which follow the existing structure.

Reply:
I'll update the descriptions in this PR, and apply the other improvements once this has been merged. I already created [an item in the board](https://github.com/orgs/hypothesis/projects/131/views/1?pane=issue&itemId=49676664) to make sure this is not lost.